### PR TITLE
Temporary dev

### PR DESCRIPTION
**Pull request by prath-dev**
Live Preview : Modal :
Screenshot from 2022-04-07 22-52-50 .
Live preview : Ratings :
Screenshot from 2022-04-07 22-55-35 .
Live preview : grids :
Screenshot from 2022-04-07 22-57-05

Screenshot from 2022-04-07 22-59-20
Live preview : Slider :
Screenshot from 2022-04-07 23-00-01

### CODE REVIEW

**Review by Balakumar3018 — APPROVED** [review]
Modal button is not working.
Otherwise all okay and good to merge

**Comment by sudosohan** [comment]
Not showing the modal demo on clicking the "click me" button

File: components/Grids/grids.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.grid-container {
+    display: grid;
+    justify-content: left;
+    align-items: left;
+    width: 62rem;
+    border: 2px solid red;
```
Comment (jatinmore) [inline components/Grids/grids.css:6]: you can use variable for colors here

Comment (prath-dev) [reply components/Grids/grids.css:6]: okkk

File: components/Grids/grids.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    grid-template-columns: 14rem 14rem 14rem;
```
Comment (sudosohan) [inline components/Grids/grids.css:7]: In such cases you can use repeat(3 , 14rem)

File: components/Grids/grids.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.items2 {
+    height: 14rem;
+    width: 22rem;
+    margin-left: 0.5rem;
+}
+
+.image {
+    width: 15rem;
```
Comment (sudosohan) [inline components/Grids/grids.css:40]: here you can use object-fit instead of giving height and width I think

**Review by sohan9819 — APPROVED** [review]
Rest everything seems good to me . Just fix the modal demo  in the docs

**Review by jatinmore — APPROVED** [review]
few suggestions:
-add font style for headings
-add examples and code snippets in the center of the div
-text content should be formatted

**Pull request merged**